scripts/generate-clients.mjs

```diff
@@ -4,21 +4,12 @@ import * as k from "kinobi";
 import { rootNodeFromAnchor } from "@kinobi-so/nodes-from-anchor";
 import { renderVisitor as renderJavaScriptVisitor } from "@kinobi-so/renderers-js";
 import { renderVisitor as renderRustVisitor } from "@kinobi-so/renderers-rust";
-import { getAllProgramIdls, getRustfmtToolchain, getToolchainArg } from "./utils.mjs";
+import { getAllProgramIdls, getToolchainArgument } from "./utils.mjs";
 
 // Instanciate Kinobi.
 const [idl, ...additionalIdls] = getAllProgramIdls().map(idl => rootNodeFromAnchor(require(idl)))
 const kinobi = k.createFromRoot(idl, additionalIdls);
 
-const ciDir = path.join(__dirname, "..", "ci");
-
-// Update programs.
-kinobi.update(
-  k.updateProgramsVisitor({
-    "featureGateProgram": { name: "featureGate" },
-  })
-);
```

Comment:
Was there any reason for changing that renaming? IIRC, having `Program` as a suffix may result in generated helpers finishing with like `ProgramProgram`.

Reply:
You're right. It wasn't doing anything here because it was missing the `solana` prefix, but I did notice at least one `PROGRAM_PROGRAM` in the generated JS client. I've updated this config!